dagmc_stats.py

```diff
@@ -201,3 +201,43 @@ def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
         t_a_r.append(top/bottom)
 
     return t_a_r
+
+def get_area_triangle(my_core, meshset):	
+    """
+    Gets the triangle area (according to the equation: sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2)
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+    
+    outputs
+    -------
+    area : (list) the triangle areas for all triangles in the meshset
+    """
+    
+    area = []
+    tris = my_core.get_entities_by_type(meshset, types.MBTRI)
+
+    for triangle in tris:
+        side_lengths = []
+        s = 0
+        coord_list = []
+
+        verts = list(my_core.get_adjacencies(triangle, 0))
+
+        for vert in verts:
+            coords = my_core.get_coords(vert)
+            coord_list.append(coords)
+
+        for side in range(3):
+            side_lengths.append(np.linalg.norm(coord_list[side] - coord_list[side - 2]))
+            # The indices of coord_list includes the "-2" because this way each side will be matched up with both
+            # other sides of the triangle (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
+
+        s = sum(side_lengths)/2
+        s = np.sqrt(s * np.prod(s - side_lengths))
+        area.append(s)
+        # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
```

Comment:
This comment should be located where `s` is calculated (just before or at the end of the line) not after the append...

Reply:
And it is already in the method doctoring, so might not be required here :)